aws/resource_aws_dx_gateway_association_proposal.go

```diff
@@ -18,12 +19,29 @@ func resourceAwsDxGatewayAssociationProposal() *schema.Resource {
 			State: schema.ImportStatePassthrough,
 		},
 
+		CustomizeDiff: customdiff.Sequence(
```

Comment:
Accepting the proposal with overridden prefixes changes the returned `RequestedAllowedPrefixesToDirectConnectGateway` value.
We only want to force a new resource if this value changes and the current proposal state is `requested`.

Reply:
This would be helpful to add as a comment within the code base.

Reply:
Will do (once the changes are made in https://github.com/terraform-providers/terraform-provider-aws/pull/8528).